src/main/Duke/virtue/Virtue.java

```diff
@@ -0,0 +1,81 @@
+package virtue;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Virtue {
+    // The scanner the chatbot uses to scan users' inputs.
+    Scanner sc = new Scanner(System.in);
+
+    // The task list to be used by the chatbot.
+    VirtueTaskList taskList = new VirtueTaskList();
+
+    // A horizontal line.
+    private static final String horizontalLine = "____________________________________________________________";
```

Comment:
its very good that you made this final, and abstracted printing with indents and printing horizontal lines, it would have been very difficult to read otherwise